pandas/compat/numpy/function.py

```diff
@@ -25,6 +25,14 @@
 from pandas.compat import OrderedDict
 
 
+class BadNumpyCall(Exception):
```

Comment:
hmm not sure I like the name. maybe ``UnsupportedFunctionCall``? and move to core.common